mappings/net/minecraft/entity/ExperienceOrbEntity.mapping

```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/class_1303 net/minecraft/entity/ExperienceOrbEntity
 	FIELD field_27009 pickingCount I
 	FIELD field_30055 DESPAWN_AGE I
+	FIELD field_30056 TICKS_PER_SECOND I
```

Comment:
This is more like an expensive update interval in ticks. Also `onEverySecond` may be named like `expensiveUpdate`? Feel free to debate.

Reply:
I think we can rename that. Oh wait, that stupid name was mine... 😛